### ISSUE TITLE
Plain text config binary

### ISSUE THREAD

**Issue by IvanMalison** [taffybar/taffybar#462]
This would help people that don't want to deal with all the complications of ghc/compilation.

**Comment by MyriaCore** [comment]
I think [phuhl/linux_notification_center](https://github.com/phuhl/linux_notification_center) uses [ConfigFile](https://www.stackage.org/lts-16.8/package/ConfigFile-1.1.4) for this. This probably wouldn't be the hardest to do, as long as we stick with a few provided widgets.

**Comment by IvanMalison** [comment]
@MyriaCore agreed, it should not be hard. otoh, I'm never going to be very motivated to do this because its just not a feature I need.
